build.js

```diff
@@ -37,17 +39,28 @@ https.get('https://uplift.shipit.staging.mozilla-releng.net/coverage/supported_e
 }).on('error', e => console.error(e.message));
 
 function makeZip(list){
```

Comment:
This part might be unnecessary if we have the list of included files. The archiver library will probably just overwrite the existing files.
Could you test?